lms/djangoapps/certificates/tests/test_queue.py

```diff
@@ -58,7 +58,10 @@ def setUp(self):
         SoftwareSecurePhotoVerificationFactory.create(user=self.user_2, status='approved')
 
     def test_add_cert_callback_url(self):
-        with patch('courseware.grades.grade', Mock(return_value={'grade': 'Pass', 'percent': 0.75})):
+        with patch(
+            'lms.djangoapps.grades.course_grades.summary',
```

Comment:
Good idea to make this more DRY.  I created a common context_manager for mocking passing grades.  Please see [this commit](https://github.com/edx/edx-platform/pull/13053/commits/56801cd8cce645c6c27d0d75d41044262cdc221d).